docs/providers/aws/guide/functions.md

```diff
@@ -402,6 +402,28 @@ functions:
     architecture: arm64
 ```
 
+## Runtime
+
+The `runtime` configuration is required and specifies which [AWS Runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) your functions use. If you set `runtime` to `provided` or `provided.al2`, include your provided runtime as a [layer](#layers). 
```

Comment:
Runtime configuration is not required (we default to `nodejs14.x` when not set in configuration)